images/fcos-bastion-image/root/usr/bin/dhcp-init.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -x
+
+mkdir -p /var/opt/{ignitions,html}
+cp ${LOCAL_PATH}/dhcp/ignitions/* /var/opt/ignitions/
```

Comment:
you added two empty file in the /usr/share/bastion_services/ignitions folder, to be copied over here.

Let's only touch the required files, the ignition will take care of adding the files in the proper final location.

Reply:
I have made the changes. Which ignition file will add these files to the correct location for its use in by butane in the same script?

Reply:
As discussed added check for the files instead of touch